Sources/Sentry/include/SentrySdkInfo.h

```diff
@@ -0,0 +1,27 @@
+#import <Foundation/Foundation.h>
+
+#import "SentrySerializable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Describes the system SDK.
+ */
+@interface SentrySdkInfo : NSObject <SentrySerializable>
+SENTRY_NO_INIT
+
+@property (nonatomic, readonly, copy) NSString *_Nullable sdkName;
```

Comment:
H: this makes sense to not be nullable: https://github.com/getsentry/sentry-cocoa/pull/626/files#r455651294